src/database/seeds/ParticipantTypes.ts

```diff
@@ -0,0 +1,28 @@
+import { Knex } from 'knex';
+import { ModelObject } from 'objection';
+import { Optional } from 'utility-types';
+
+import { ParticipantType } from '../../api/entities/ParticipantType';
+
+type ParticipantTypeModel = ModelObject<ParticipantType>;
+
+const sampleData: Optional<ParticipantTypeModel, 'id'>[] = [
+  { typeName: 'DSP' },
+  {
+    typeName: 'Advertiser',
+  },
+  {
+    typeName: 'Data Provider',
+  },
+  {
+    typeName: 'Publisher',
+  },
+];
+
+export async function seed(knex: Knex): Promise<void> {
+  // Deletes ALL existing entries
+  await knex('participants_types').del();
```

Comment:
This will break the data every time the seeds run. e.g. if there's a Publisher 'Disney' linked to participants_types.id=1, when this drops all the participants and reinserts them Disney will be linked to a non-existent type 1, and there will be a new set of participants_types (ids 5 to 8).

I'd thought of these seeds as a way to insert testing data so a new dev just getting started doesn't have to set up all of their own test data. We could create two different set of seeds - one for everything to set up base data like this (which preserves IDs to not break existing relationships) and a different set for test data.